docs/MockFunctions.md

```diff
@@ -163,7 +163,7 @@ jest.mock('axios');
 
 test('should fetch users', () => {
   const resp = {data: [{name: 'Bob'}]};
-  axios.get.mockReturnValue(Promise.resolve(resp));
+  axios.get.mockResolvedValue(resp);
```

Comment:
@danawoodman, documentation should have been changed too (e.g. `mockReturnValue` vs `mockResolvedValue`): 
```
Once we mock the module we can provide a mockReturnValue for .get that returns the data we want our test to assert against. In effect, we are saying that we want axios.get('/users.json') to return a fake response.
```

I've noticed it because create-react-app uses Jest 20 which doesn't have `mockResolvedValue` yet.

Reply:
@DamianFekete fwiw, create-react-app is on Jest 22 and these docs are for Jest 22

Reply:
@rickhanlonii, the released version of create-react-app, 1.5.2, still uses [jest@20.0.4]( https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/package.json).
The next branch (1.5.1 😄) uses Jest 22.

Reply:
Ah, good to know